lms/static/sass/theme-colors.scss

```diff
@@ -1,21 +1,2 @@
 @import 'lms-overrides';
-
-// Set default colors in case they aren't set on the parent file
-$main-color: #ff8000 !default;
-$link-color: #126f9a !default;
-$header-bg: #ffffff !default;
-$footer-bg: #ffffff !default;
-
-$btn-primary-bg: theme-color("primary") !default;
-$btn-primary-color: theme-color("inverse") !default;
-$btn-primary-border-color: theme-color("primary") !default;
-$btn-primary-hover-bg: theme-color("primary") !default;
-$btn-primary-hover-color: theme-color("inverse") !default;
-$btn-primary-hover-border-color: theme-color("primary") !default;
-
-$btn-secondary-bg: theme-color("primary") !default;
-$btn-secondary-color: theme-color("inverse") !default;
-$btn-secondary-border-color: theme-color("primary") !default;
-$btn-secondary-hover-bg: theme-color("primary") !default;
-$btn-secondary-hover-color: theme-color("inverse") !default;
-$btn-secondary-hover-border-color: theme-color("primary") !default;
+@import '../../../common/variables';
```

Comment:
Do we need this file? We could add the following to `lms/static/sass/theme-overrides.scss`
```diff
-  @import 'theme-colors';
+ @import 'lms-overrides';
+ @import '../../../common/variables';
```

Reply:
It is needed because it is also imported by `./discussion/lms-discusssion-bootstrap.scss`